docker-1_host/templates/resquev5/Dockerfile

```
@@ -17,12 +17,10 @@ COPY php.ini /etc/resque-serial/php.ini
 COPY supervisord.conf /etc/supervisord.conf
 COPY resque-5.conf /etc/supervisor.d/resque-5.conf
 COPY config-5.yml /etc/resque-serial/config-5.yml
-COPY resqu-server-5.2.1.zip /opt/qu/resqu-server-5.2.1.zip
 
-RUN cd /opt/qu && unzip /opt/qu/resqu-server-5.2.1.zip && \
-    mv resqu-server-5.2.1/* /opt/qu/php-resque-5/ && \
-    rm -rf /opt/qu/resqu-server-5.2.1.zip && \
-    cd /opt/qu/php-resque-5 && curl -s http://getcomposer.org/installer | php && \
-    cd /opt/qu/php-resque-5 && php composer.phar install --no-dev --no-plugins --no-scripts
+RUN curl -sL https://raw.github.com/QualityUnit/resqu-server/5.4.0/build/php-resque-serial-5.4.0.zip -o /opt/qu/php-resque-serial-5.4.0.zip && \
+    echo "d0db7c4e544891dd4a8da949338a34f3d8bc23756f41e162e80f3b97e872a612 /opt/qu/php-resque-serial-5.4.0.zip" | sha256sum -c && \
+    unzip /opt/qu/php-resque-serial-5.4.0.zip -d /opt/qu/php-resque-5/ && \
+    rm -rf /opt/qu/php-resque-serial-5.4.0.zip
```

```suggestion
RUN cd /opt/qu && \
    VERSION=5.4.0 && \
    curl -fLOsS https://github.com/QualityUnit/resqu-server/raw/${VERSION}/build/php-resque-serial-${VERSION}.zip && \
    echo "d0db7c4e544891dd4a8da949338a34f3d8bc23756f41e162e80f3b97e872a612  php-resque-serial-${VERSION}.zip" | sha256sum -c && \
    unzip php-resque-serial-${VERSION}.zip -d php-resque-5 && \
    rm php-resque-serial-${VERSION}.zip
```

Please, modify it everywhere else too.

---

Should not we tie checksum to version? I believe that if we define version like this we should also place checksum there.

---

Do you mean declaring a variable for the hash too? What's the point if there's just a single occurrence? The hash is fundamentally linked to the version, so I'm unsure of what you mean.

---

Yes I meant to declare that this version have this hash. Anyway, it is true that there is only a single occurrence so it is probably not necessary. I will therefore test and change this everywhere.

---

Changed everywhere via https://github.com/QualityUnit/LiveAgent-Docker/pull/6/commits/7ad46e3d20927ccbfd28d3d8f77e1641275121b4

---

> Please, be so kind and don't close other's discussions.

The discussions was closed after I applied your suggestion. I did not resolved that manually.